test/cmd/describer.sh

```diff
@@ -33,5 +33,16 @@ os::cmd::try_until_success 'eventnum=$(oc get events | wc -l) && [[ $eventnum -g
 # resources without describers get a default
 os::cmd::expect_success_and_text 'oc describe events' 'Namespace:	cmd-describer'
 
+# TemplateInstance
+os::cmd::expect_success 'oc create -f test/extended/testdata/templates/templateinstance_objectkinds.yaml'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'FOO:	3 bytes'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'Name:		templateinstance'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'Namespace:	cmd-describer'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'Type:		Ready'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'Status:		True'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'Secret:	testing/secret'
```

Comment:
Running it here, it appears that you need s/testing/cmd-describer/ on these lines

Reply:
Yep, wasn't done yet, pushed it so that you could test it